src/ch/blinkenlights/android/vanilla/PlaybackService.java

```diff
@@ -2010,8 +2037,8 @@ public void onAudioFocusChange(int type)
 		Log.d("VanillaMusic", "audio focus change: " + type);
 		switch (type) {
 		case AudioManager.AUDIOFOCUS_LOSS_TRANSIENT_CAN_DUCK:
-			mDuckedLoss = (mState & FLAG_PLAYING) != 0;
-			unsetFlag(FLAG_PLAYING);
+			mDuckedLoss = true;
```

Comment:
Yeah, I guess I thought it was redundant. There's no harm in doing it, I'll add it back in in case it fixes some obscure/platform specific bug.